src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -1769,11 +1769,6 @@ protected internal virtual TypeDefinition MarkType (TypeReference reference, Dep
 
 			MarkSerializable (type);
 
-			// This marks static fields of KeyWords/OpCodes/Tasks subclasses of an EventSource type.
```

Comment:
I think we can't physically delete the support for this at this point - we probably need to keep it under `if (_context.GetTargetRuntimeVersion () < TargetRuntimeVersion.NET6)` same as #1692 did with the serializer support.

Reply:
Thanks! I'm not very familiar with the conditions where the linker runs in old runtime versions. 

What does this mean with this fix on types with the annotations? Is it safe to assume that pre-NET6 versions would not have annotations on types?